index.html

```diff
@@ -5,8 +5,221 @@
+        </ul>
+      </section>
+      <section>
+        <h2>Категории товаров</h2>
+        <ul>
+          <li><a href="#">Виртуальная реальность</a></li>
+          <li><a href="#">Моноподы для селфи</a></li>
+          <li><a href="#">Экшн-камеры</a></li>
+          <li><a href="#">Фитнес-браслеты</a></li>
+          <li><a href="#">Умные часы</a></li>
+          <li><a href="#">Квадрокоптеры</a></li>
+        </ul>
+      </section>
+      <section>
+        <h2>Наши услуги</h2>
+        <dl>
+          <dt>Доставка</dt>
+          <dd>
+            <h3>Доставка</h3>
+            <p>
+              Мы с удовольствием доставим ваш товар прямо к вашему<br>
+              подъезду совершенно бесплатно! Ведь мы неплохо<br>
+              заработаем, поднимая его на ваш этаж.
+            </p>
+          </dd>
+          <dt>Гарантия</dt>
+          <dd>
+            <h3>Гарантия</h3>
+            <p>
+              Если из-за возгорания купленного у нас товара у вас сгорит дом — не переживайте, мы выдадим вам новый.<br>
+              Товар, не дом, конечно же.
+            </p>
+          </dd>
+          <dt>Кредит</dt>
+          <dd>
+            <h3>Кредит</h3>
+            <p>
+              Залезть в долговую яму стало проще! Кредитные консультанты подберут для вас наиболее выгодные<br>
+              условия кредита. Выгодные для нашего банка, разумеется.
+            </p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h2>Представленные у нас бренды</h2>
+        <ul>
+          <li><img alt="Логотип компании DJI" height="100" src="img/brands/dji.png" width="260"></li>
+          <li><img alt="Логотип компании SP GADGETS" height="100" src="img/brands/sp-gadgets.png" width="260"></li>
+          <li><img alt="Логотип компании Go Pro" height="100" src="img/brands/go-pro.png" width="260"></li>
+          <li><img alt="Логотип компании VIVE" height="100" src="img/brands/vive.png" width="260"></li>
+        </ul>
+      </section>
+      <section>
+        <h2>О нас</h2>
+        <p>
+          Огромный выбор гаджетов не оставит равнодушным гика,<br>
+          который есть в каждом из нас.
+        </p>
+        <p>Мы можем доставить ваш товар в самые отдаленные точки России! DEVICE работает со многими транспортными
+          компаниями:</p>
+        <ul>
+          <li>Деловые линии</li>
+          <li>Автотрейдинг</li>
+          <li>ЖелДорЭкспедиция</li>
+        </ul>
+        <a href="#">Подробнее о нас</a>
+      </section>
+      <section>
+        <h2>Контакты</h2>
+        <p>
+          Вы можете забрать товар сами, заехав в наш офис. Заодно, вы сможете<br>
+          проверить работоспособность покупки. Всякое бывает.
+        </p>
+        <!-- Здесь будет карта. -->
+        <a href="#">Напишите нам</a>
+      </section>
+    </main>
 
+    <footer>
+      <a><img alt="Логотип магазина DEVICE" height="35" src="img/logo-footer.png" width="163"></a>
+      <div>
```

Comment:
можно использовать p для группировки фразового контента
```suggestion
      <p>
```

Reply:
Согласен. Лучше поменяю на тег p.